index.bs

```diff
@@ -163,6 +163,38 @@ spec: RFC8941; urlPrefix: https://datatracker.ietf.org/doc/html/rfc8941#
     to use the Geolocation API, but any other ports on "<code>https://example.com</code>"
     could use it too.</p>
   </div>
+  <div class="example">
+    <p>JSPlaygroundCorp Inc. wants to host user-generated web applications, but wants the
+    browser to manage their permissions to use powerful features in isolation of each other.
```

Comment:
```suggestion
    browser to manage their permissions to use [=powerful features=] in isolation of each other.
```

Reply:
browser to manage their permissions to use [=powerful features=] in isolation of each other.